doc/psud/PSU_daemon_design.md

```diff
@@ -168,14 +270,53 @@ class PsuBase(device_base.DeviceBase):
 	
     def get_input_current(self):
         raise NotImplementedError
-    ...
 
+    def get_psu_power_warning_threshold(self)
```

Comment:
can we have a set API for user to override?

Reply:
I do not suggest having an API to override the thresholds. As discussed before, it should be controlled by the platform vendor only because users do not have full knowledge to decide what the threshold should be. For example, in our system, the thresholds depend on temperatures as well.